lib/src/rules/curly_braces_in_flow_control_structures.dart

```diff
@@ -89,6 +89,11 @@ class _Visitor extends SimpleAstVisitor {
   void visitIfStatement(IfStatement node) {
     var elseStatement = node.elseStatement;
     if (elseStatement == null) {
+      if (node.parent is IfStatement &&
+          node == (node.parent! as IfStatement).elseStatement) {
```

Comment:
Our preferred style is to assign fields like this to a local variable so that type inference can do more of the work. In this case that would look like
```dart
var parent = node.parent;
if (parent is IfStatement && node == parent.elseStatement) {
  // ...
}
```
That allows both the null check (`!`) and the cast to be omitted.

Reply:
Thanks for your feedback. I will change it when I return back, and will add a line in`changes`.